keycloak/themes/hmda/login/messages/messages_en.properties

```diff
@@ -24,3 +24,7 @@ invalidPasswordNotUsernameMessage=&#8226; Invalid password: must not be equal to
 invalidPasswordRegexPatternMessage=&#8226; Invalid password: fails to match regex pattern(s).
 invalidPasswordHistoryMessage=&#8226; Invalid password: must not be equal to any of last {0} passwords.
 invalidPasswordGenericMessage=&#8226; Invalid password: new password doesn''t match password policies
+
+# custom for hmda
+# DO NOT COMMIT UPDATES TO OR REMOVE THE "support@localhost.localdomain"
+hmdaEnterEmailAddress=To register as a filer for more than one institution, you may select multiple institutions from the list of available institutions based on your email domain. If one or more institutions is not available in the list based on your email domain, contact support@localhost.localdomain for assistance with registering for the additional institution(s).
```

Comment:
In theoryland, it should just work.  All attributes in [`hmda/login/theme.properties`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/themes/hmda/login/theme.properties) should be available in the templates with the `properties` object.

Unfortunately, I don't think there really is a guide.  I think thing `msg` function is a freemark util cooked up by Keycloak.  But, there are example of passing values into templatized attributes:

* [`hmda/login/register.ftl:4`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/themes/hmda/login/register.ftl#L4)